### ISSUE TITLE
Remove the outdated information about lighting balanaces

### ISSUE THREAD

**Issue by pavlenex** [btcpayserver/btcpayserver-doc#1456]
@dennisreimann can you confirm this is outadted?

<img width="1113" alt="Screenshot 2024-11-18 at 13 14 01" src="https://github.com/user-attachments/assets/ac8e4f04-08c5-47b4-9a74-fb840f563583">

https://docs.btcpayserver.org/Dashboard/#dashboard

I am not sure I understand what it means tbh, as afaik we do show all info for lighnting in the dashboard?

**Comment by dennisreimann** [comment]
Yes, with the new v2 versions we provide a balance graph and general channel info. Imho we can remove that "tip".
